docs/advanced/node-local-storage.md

```diff
@@ -0,0 +1,356 @@
+```
+
+You can also connect to the pod and write to the volume which is mounted on
+`/usr/test-pod`, you will see in the host on `/mnt`.
+
+If you delete a pod, as it is part of this Stateful Set, it will be recreated and
+continue to use the volume.
+
+What if you want to delete this application and use the PVs for a different
+application?
+
+Basically, as the retain policy is set to `Retain` (just to be safe), you need
+to do some manual operations documented [here][1]. Note that using the `Delete`
+policy this is different, so you may want to choose the policy that better fits
+your use case.
+
+[1]: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#retain
+
+### Automatic Local Persistent Volume creation and handling
+
+The example manually created the `Kind: PersistentVolume` for each node. Also,
+the PV will continue to exist even if the PVC is deleted and, therefore, the
+lifecycle of the PV needs to be handled manually.
+
+Those limitation don't scale well with the number of nodes and applications,
+for that reason an external provisioner that will automatically create those
+objects when nodes are available and manage the lifecycle of the volume was
+created by sig-storage:
+	https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/
+
+### Limitations
+
+There are some limitations you might want to have in mind:
+
+1. Lokomotive Kubernetes currently only exposes `/mnt/` to the kubelet, so only that
+directory can be used for node local storage. You can **manually** mount different
+devices on `/mnt/sda` or paths inside `/mnt` at your convenience.
+
+1. Using `setup_raid = "true"` will create a single RAID 0 array with all
+the spare disks and mount it in `/mnt/`. Therefore all the node storage will be
+available to only one PV, thus it will be available to only one PVC and one pod.
+This is the only automated setup supported in Lokomotive at the time of this
+writing.
+
+1. You can share the volume in different PVs if you create a sub-directory per
+PV, bind mount each of them and only expose the subdirectories. It is explained
+in detail [here][1]. But take into account that it needs to be done manually.
+
+1. When using `setup_raid = "true"` the filesystem is `ext4` and there is no
+way to use different one right now.
+
+[1]: https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/operations.md#sharing-a-disk-filesystem-by-multiple-filesystem-pvs
+
+### Operation and FAQs
+
+* [Recommendations and tips on how to operate][1]. Bear in mind, however, that the provisioner is not being used in Lokomotive
+right now.
+
+ * You may also want to check out the [FAQs][2].
+
+[1]: https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/operations.md
+[2]: https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/faqs.md
+
+### Useful links
```

Comment:
I am not sure if rest of this document should be here, rather than helping user it is just confusing user to follow lot of other links.

I would rather let this document be only something we support and have tried, tested and trusted.

Reply:
Okay... But I think that is useful and I would prefer to have this somewhere, where do you suggest? Or you suggest to just drop this?